dask/delayed.py

```diff
@@ -248,9 +254,13 @@ def delayed(obj, name=None, pure=False, nout=None):
     if isinstance(obj, Delayed):
         return obj
 
-    task, dasks = to_task_dasks(obj)
+    if isinstance(obj, base.Base) or traverse:
+        task, dasks = to_task_dasks(obj)
+    else:
+        task = quote(obj)
+        dasks = []
```

Comment:
If `obj` is a tuple that looks like a task, or a list containing tuples that look like tasks. Looks like the second isn't handled though, will fix.